Improve Design Of the Project

saurabhsharma104 opened the pull request.

saurabhsharma104 committed:
Improve Design Of the Project

yogesh2104 commented on index.html:
```diff
@@ -169,10 +169,10 @@ <h1 class="justify-content-center d-flex">STUDY MATERIAL</h1>
-    <iframe style="border:none" max-width="25%" height="550" src="https://whimsical.com/embed/Q5Eko8fLLZZhMMYWgtsPD4"></iframe>
-     <iframe style="border:none" max-width="25%" height="550" src="https://whimsical.com/embed/8NN72vuyqAbb95oaEoSzCn"></iframe>
-     <iframe style="border:none" max-width="25%" height="550" src="https://whimsical.com/embed/GeaRMSWME9gsidDrS3CRBg"></iframe>
+    <iframe style="border:solid" max-width="25%" height="550" src="https://whimsical.com/embed/14qC8Nr3dhY3Leyn5KwabC"></iframe>
```
make a button for every road map and open it in new tap

yogesh2104 reviewed (changes requested):
improve design

saurabhsharma104 committed:
Improve design and add some new tools 

Improve the design and add some new tools for a better website.

Now User takes more content here.

yogesh2104 reviewed (commented):
Good work
